bin/data-push.js

```diff
@@ -90,11 +90,11 @@ Promise.resolve().then(async () => {
       },
       schedule: argv.schedule
     }
-    await datahub.push(dataset, options)
-
+    let res = await datahub.push(dataset, options)
+    let revisionId = res.flow_id.split('/').pop()
```

Comment:
Can not you do `a, b, c = string.split(' ')`  in Javascript BTW? 
Eg 
```
let owner, dataset, revisionId = res.flow_id.split('/')
```
and you would not need all that config staff below

Reply:
Here as well - `const` not let

Reply:
we do not need, since `owner` is `ownerid` which comes from specstore, but config owner is `username`.